test/corecel/cont/Span.test.cc

```diff
@@ -253,6 +255,80 @@ TEST(SpanTest, io_manip)
         EXPECT_EQ("{123   ,456  ,789  }", os.str());
     }
 }
+TEST(SpanTest, ldg_span)
+{
+    {
```

Comment:
Just a minor thing, but how about breaking this test into two `TEST(LdgSpanTest, pod)` and `TEST(LdgSpanTest, opaque_id)` instead of two blocks in one test?